.github/workflows/notify-on-review-wanted.yml

```
@@ -0,0 +1,37 @@
+name: Notify on Review Wanted
+on:
+  issues:
+    types: [labeled]
+  pull_request_target:
+    types: [labeled]
+
+permissions:
+  contents: read
+
+jobs:
+  notifyOnReviewWanted:
+    name: Notify on Review Wanted
+    if: github.repository == 'nodejs/node'
```

```suggestion
    if: github.repository == 'nodejs/node' && github.event.label == 'review wanted'
```

This part is really important...